lib/constants/constants.dart

```diff
@@ -0,0 +1,5 @@
+const titleFontSize = 38.0;
+const subtitleFontSize = 24.0;
+const secondSubtitleFontSize = 18.0;
+const logoBlue = 0xff264653;
+const coolGreen = 0xff2a9d8f;
```

Comment:
What is this? Can it be named more suggestive?

Reply:
Done